lib/lita/handlers/down_for_everyone.rb

```diff
@@ -1,4 +1,5 @@
 require "lita"
+require "pry"
```

Comment:
Would you mind removing this dependency? It doesn't look like it's actually used anywhere (though I bet you used it when you were debugging).